src/components/Item.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { observer } from 'mobx-react';
+
+const Item = observer((props) => {
+    const item = props.item;
```

Comment:
Instead, you can destructure the props in the definition of the function, then it's a lot clearer what the props are.

`const Item = observer(({ item }) => {...`